src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -284,8 +294,18 @@ private StatementSyntax MutateSubExpressionWithIfStatements(StatementSyntax orig
 
         private SyntaxNode MutateWithConditionalExpressions(ExpressionSyntax currentNode)
         {
-            var expressionAst = currentNode;
-            foreach (var mutant in FindMutantsRecursive(currentNode))
+            var nodeToTracks = new List<SyntaxNode>(currentNode.ChildNodes());
+            nodeToTracks.Add(currentNode);
+            var expressionAst = currentNode.TrackNodes(nodeToTracks);
```

Comment:
I think this works as well: 

```suggestion
            var expressionAst = currentNode.TrackNodes(currentNode.ChildNodes().Append(currentNode));
```

No need for the new list then.

Reply:
ok